packages/govuk-frontend/src/govuk/components/summary-list/_index.scss

```diff
@@ -172,6 +172,8 @@
 
   // Additional block for the summary card
   .govuk-summary-card {
+    @include govuk-font($size: 19, $weight: bold);
```

Comment:
Can we drop this too?

We style `.govuk-summary-card__title` and `.govuk-summary-list` etc separately

```suggestion
```

Reply:
To clarify, are you suggesting moving the `govuk-font` declaration to the `__title` element?

Reply:
Yeah, keeping it on `__title` elements (no `font-family`) and remove it everywhere else

Then when https://github.com/alphagov/govuk-frontend/issues/4277 is picked up we'd naturally remove it from `__title` elements too?

Otherwise I'm totally happy moving it to the component block for safety but it's not really necessary

Reply:
Nope, makes sense!